src/System.IO.FileSystem/src/System.IO.FileSystem.builds

```diff
@@ -10,22 +10,13 @@
     </Project>
     <Project Include="System.IO.FileSystem.csproj">
```

Comment:
I have left these removed from src\*.builds file, but kept the ns1.3/netcore50 stuff in the csproj and project.json for the package builds under ns1.3/netcore50.